modules/hcp-np-capacity-blocks.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+// hosted_control_planes/hcp-manage/hcp-manage-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-np-capacity-blocks_{context}"]
+= Configuring node pool capacity blocks on {aws-short}
+
+After creating a hosted cluster, you can configure node pool capacity blocks for graphics processing unit (GPU) reservations on {aws-first}.
+
+.Procedure
+
+. Create GPU reservations on {aws-short} by running the following command:
++
+[IMPORTANT]
+====
+The zone of the GPU reservation must match your hosted cluster zone.
+====
++
+[source,terminal]
+----
+$ aws ec2 describe-capacity-block-offerings \
+      --instance-type "p4d.24xlarge"\ #<1>
+      --instance-count  "1" \ #<2>
+      --start-date-range "$(date -u +"%Y-%m-%dT%H:%M:%SZ")"  \ #<3>
+      --end-date-range "$(date -u -d "2 day" +"%Y-%m-%dT%H:%M:%SZ")" \ #<4>
+      --capacity-duration-hours 24 \ #<5>
+      --output json
+----
+<1> Defines the type of your {aws-short} instance, for example, `p4d.24xlarge`.
+<2> Defines your instance count, for example, `1`. You can add up to `20` instance counts. //check max count
```

Comment:
the max count is `64`
I saw the info in aws console when created capacity block
```
The instance purchase quantity must be an integer value between 1 and 64 instances.
```